portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -12,7 +12,10 @@ public class HelloWorldServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+      response.setContentType("text/html;");
```

Comment:
There's some extra whitespace and indentation that could be removed in this file (blank lines, and the spacing of the body of the `doGet` function.

Reply:
Done